src/psp2_ui.cpp

```diff
@@ -47,14 +58,30 @@ AudioInterface& Psp2Ui::GetAudio() {
 
 int _newlib_heap_size_user = 192 * 1024 * 1024;
 
+vita2d_shader* shaders[SHADERS_NUM];
+char* shader_names[] = {
+	"None",
+	"Sharp Bilinear",
+	"LCD 3x",
+	"xBR"
+};
```

Comment:
If the reason for extern are strange linker errors: Declare these variables as "static"